source/faq.rst

```diff
@@ -1,264 +1,219 @@
+.. _alertmanager:
+.. _how-to-setup-alerting-with-grafana:
 
-.. note:: Consider increasing minimum resolution
-   when |pmm-server| and |pmm-client| are on different networks,
-   or when :ref:`pmm.amazon-rds`.
+********************************
+How do I set up Alerting in PMM?
+********************************
 
-.. _alertmanager:
+When a monitored service metric reaches a defined threshold, PMM Server can trigger alerts for it either using the Grafana Alerting feature or by using an external Alertmanager, a high-performance solution developed by the Prometheus project to handle alerts sent by Prometheus.
+(Both are advanced features that require knowledge of third-party tools.)
 
-How to set up Alerting in PMM?
-================================================================================
+With these methods you must configure alerting rules that define conditions under which an alert should be triggered, and the channel used to send the alert (e.g. email).
 
-You can make PMM Server trigger alerts when your monitored service reaches some thresholds in two ways:
+Grafana Alerts are already integrated into PMM Server and may be simpler to get set up.
 
-* using `Grafana Alerting feature <https://grafana.com/docs/grafana/latest/alerting/rules/>`_,
-* using external `Alertmanager <https://github.com/prometheus/alertmanager>`_ (a
-  high-performance solution developed by the Prometheus project to handle alerts sent
-  by Prometheus).
+Alertmanager allows the creation of more sophisticated alerting rules and can be easier to manage installations with a large number of hosts. This additional flexibility comes at the expense of simplicity and requires advanced knowledge of Alertmanager rules.
 
-Both options can be considered advanced features and require knowledge of
-third-party documentation.
+Alerting in Grafana allows attaching rules to your dashboard panels.
 
-Either with Grafana Alerting or with Alertmanager you need to configure some
-alerting rule to define conditions under which the alert should be triggered,
-and the channel used to send the alert (e.g. email).
+.. note::
 
-Grafana Alerts are already integrated into PMM Server and may be simpler to get set up,
-while Alertmanager allows the creation of more sophisticated alerting rules and
-can be easier to manage installations with a large number of hosts; this
-additional flexibility comes at the expense of simplicity and requires advanced
-knowledge of Alertmanager rules. Currently Percona cannot offer support for
-creating custom rules so you should already have a working Alertmanager instance
-prior to using this feature, however we are working hard to bring an integrated
-Alertmanager solution to make rule generation easy!
+   Percona cannot offer support for creating custom rules so you should already have a working Alertmanager instance prior to using this feature. (However, we are working hard to bring an integrated Alertmanager solution to make rule generation easy.)
```

Comment:
We actually do offer custom rules if you are a Percona Customer. I would explicitly call this out as it is a feature of our offering.